configure.ac

```diff
@@ -94,7 +94,7 @@ AC_SUBST([LIBFLUX_HOSTLIST_VERSION_INFO])
 ##
 # Checks for programs
 ##
-AC_PROG_CC_C99
```

Comment:
This is obsolete but it may be the way that you forced the RHEL7 compilers into C99 mode.